packages/components/src/session.tsx

```diff
@@ -1,137 +0,0 @@
-import { Signer } from '@ethersproject/abstract-signer'
-import {
-  ISessionContext,
-  SessionContext,
-  useAuthenticate,
-  useEagerConnect,
-  useSigner,
-} from '@nft/hooks'
-import { useWeb3React } from '@web3-react/core'
-import decode, { JwtPayload } from 'jwt-decode'
-import React, { FC, useCallback, useEffect, useMemo } from 'react'
-import { useCookies } from 'react-cookie'
-
-export const COOKIE_ADDRESS = 'address'
-export const COOKIE_JWT_TOKEN = (address: string): string =>
-  `jwtToken-${address.toLowerCase()}`
-
-const cookieOptions = {
-  secure: true,
-  sameSite: true,
-  path: '/',
-}
-
-const currentJWT = (cookies: {
```

Comment:
Cookies helper functions moved to the application side in the `session.ts` file that controls where to store the cookie and how to decode it